package.json

```diff
@@ -53,18 +53,19 @@
     "url": "https://github.com/onefinestay/react-daterange-picker"
   },
   "peerDependencies": {
-    "react": "^0.14.0 || 15.x.x",
```

Comment:
I wonder if it's possible to support react 14/15 still? From what I can see, most of the changes in this PR are related to test changes 🤔

Reply:
Good point, it is mostly due to the test utils addons. I'll check about backwards compatibility, one moment.

Reply:
Hi @AlanFoster,

According to https://stackoverflow.com/a/43669552/1885857 These changes require 15.5+. 

You might be able to do conditional imports / code execution based on react version?

May I ask if there is a strong demand for supporting 14 and 15 (before 15.5)?

Alternatively, if there is a legitimate demand, could a separate branch be set up and an alternate npm version / project be allocated?

Reply:
@james-ff The changes in this PR require react 15.5+ from the perspective of our tests, but the React component itself could still work on React 15. Note that we specify React as a peer dependency, so this may still be backwards compatible. If this is the case, I can release this PR as a minor version.

> May I ask if there is a strong demand for supporting 14 and 15 (before 15.5)?

I can't be sure of all of our user's React versions, however I know with our large codebase we've just landed on 15.6

> Alternatively, if there is a legitimate demand, could a separate branch be set up and an alternate npm version / project be allocated?

I can release a major version if need be, just confirming if it's needed or not :+1:

Reply:
That would be great if you could create a minor version that supports 16 =]

Reply:
As long as I can start using the module via npm

Reply:
@AlanFoster is minor version best?

Reply:
@james-ff Sorry for the long delay; I've confirmed that this will work with 14 and 15 still, can we change this to support 14, 15, 16 please :)

Reply:
It was just a simple syntax change in the end :)

https://github.com/onefinestay/react-daterange-picker/commit/4f61f2f6d9eca3832a1f68cdbdf85189d9f19dbe#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R57